pkg/crc/ssh/client.go

```diff
@@ -72,16 +72,14 @@ func clientConfig(user string, keys []string) (*ssh.ClientConfig, error) {
 }
 
 func (client *NativeClient) session() (*ssh.Session, error) {
-	if client.conn == nil {
-		var err error
-		config, err := clientConfig(client.User, client.Keys)
-		if err != nil {
-			return nil, fmt.Errorf("Error getting config for native Go SSH: %s", err)
-		}
-		client.conn, err = ssh.Dial("tcp", net.JoinHostPort(client.Hostname, strconv.Itoa(client.Port)), config)
-		if err != nil {
-			return nil, err
-		}
+	var err error
+	config, err := clientConfig(client.User, client.Keys)
+	if err != nil {
+		return nil, fmt.Errorf("Error getting config for native Go SSH: %s", err)
+	}
+	client.conn, err = ssh.Dial("tcp", net.JoinHostPort(client.Hostname, strconv.Itoa(client.Port)), config)
+	if err != nil {
+		return nil, err
```

Comment:
I guess this reverts `81268bf2 Connect ssh only once, and reuse the same connection`, not clear from the log why the error handling can't be fixed instead of a revert.